internal/visualization/console/reconciler.go

```diff
@@ -161,16 +161,28 @@ func (r *Reconciler) mutateConsolePlugin() error {
 			BasePath:  "/",
 			Port:      r.pluginBackendPort(),
 		},
-		Proxy: []consolev1alpha1.ConsolePluginProxy{{
-			Type:      "Service",
-			Alias:     "backend",
-			Authorize: true,
-			Service: consolev1alpha1.ConsolePluginProxyServiceConfig{
-				Name:      r.LokiService,
-				Namespace: r.Namespace(),
-				Port:      r.LokiPort,
+		Proxy: []consolev1alpha1.ConsolePluginProxy{
+			{
+				Type:      "Service",
+				Alias:     "backend",
+				Authorize: true,
+				Service: consolev1alpha1.ConsolePluginProxyServiceConfig{
+					Name:      r.LokiService,
+					Namespace: r.Namespace(),
+					Port:      r.LokiPort,
+				},
+			},
+			{
+				Type:      "Service",
+				Alias:     "korrel8r",
+				Authorize: true,
```

Comment:
This should be `false` as POST request will be blocked by the console proxy without a `X-CSRFToken` header, which is the current behavior.
```suggestion
				Authorize: false,
```